scripts/cloudfoundry/bat-cdn/bat-cdn.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+
+require 'yaml'
+require 'json'
+
+_env= (ARGV)[0]
+
+input = YAML.load_file('cdn-config.yml')
+
+input =  input["#{_env}"].to_json
+
+puts input
+
+if __FILE__ == $0
+
+cmd = "cf update-service bat-cdn-#{_env} -c '" + "#{input}".gsub(" ",',') + "'"
```

Comment:
You may want to try the `.join(",")` function to create a comma delimited string from a list. Before translating to json.